tox.ini

```diff
@@ -13,4 +19,4 @@ deps =
     django220: Django>=2.2.0,<2.3.0
     django300: Django>=3.0.0,<3.1.0
     django310: Django>=3.1.0,<3.2.0
-    djangolatest: https://github.com/django/django/archive/master.tar.gz
```

Comment:
Can we keep the djangolatest? It's useful for us to know beforehand if something will break in the next Django release.

Reply:
Sure - it just didn't work when I ran the action locally with act. I'll give it another try 😉

Reply:
@fjsj it's back in - although it does fail ;)